.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "assets"]
+	path = assets
+	url = https://github.com/kivutar/ludo-assets.git
```

Comment:
This looks right...... One thing you can do is add `branch=master` tags so that it's easy to update, but this works for now. :+1: